docs/changelogs/v2.3.1.md

```diff
@@ -0,0 +1,46 @@
+Here's the changelog with improved formatting and clarity. The changes focus on readability and structure.
+
+---
+
+## Changelog
+
+### Important Features
+
+#### New Builtin Icons Page
+
+- **URL**: `CODER_ACCESS_URL/icons` e.g., `https://coder.mycompany.com/icons`
+  ![Builtin icons page](../images/icons-gallery.png)
+
+#### Icon Selector in Emoji Picker
+
+![Icon selector](https://user-images.githubusercontent.com/418348/273346591-53837fd8-f3ce-4a71-b73e-bda9a62bb7c3.png)
+
+**Impact**: Both features improve usability in finding and using builtin icons.
+**Docs**: [Icons Documentation](https://coder.com/docs/v2/latest/templates/icons)
+
+### Features
+
+- **Expose User Seat Limits**: Prometheus metrics (#10169) (@mtojek)
+- **Control Autostart Days**: For templates (#10226, #10263) (@Emyrk)
+- **Batched Agent Metadata**: Add support (#10223, #10224) (@mafredri)
+- **Helm Chart Update**: ExtraTemplates in provisioners (#10256) (@johnstcn)
+- **Workspace Filtering**: Limit to `Running`, `Stopped`, `Failed`, `Pending` (#10283) (@BrunoQuaresma)
+
+### Bug Fixes
+
+- **Screen Encoding**: Fixes an issue with web terminal rendering by using UTF-8 (#10190) (@code-asher)
+- **Workspace Names**: Display template names even if no display name is set (#10233) (@Parkreiner)
+- **API Context**: Pass through to provisionerdserver (#10259) (@johnstcn)
```

Comment:
Perhaps we categorize these under "other" since the category `API Context` means nothing to our users

Reply:
we can remove this. as it is not a user-facing change.